scrc/enums/section.py

```diff
@@ -13,4 +13,9 @@ class Section(Enum):
 
     @classmethod
     def without_facts(cls):
-        return cls.HEADER, cls.HEADER,cls.CONSIDERATIONS, cls.RULINGS, cls.FOOTER
\ No newline at end of file
+        return cls.HEADER, cls.HEADER,cls.CONSIDERATIONS, cls.RULINGS, cls.FOOTER
```

Comment:
Does the topic need to be added here?
And does it make sense to reformulate it as a subtraction? (all sections minus the facts)

Reply:
As far as I can tell, this function was not used anywhere in the project so I removed it. With the new implementation, you don't need to define a facts section anyway if there isn't one.